ui/apps/platform/src/Containers/Violations/Details/Deployment/ContainerConfiguration.tsx

```diff
@@ -18,7 +33,11 @@ function ContainerConfiguration({ deployment }: ContainerConfigurationProps): Re
         content = deployment.containers.map((container, i) => (
             <React.Fragment key={container.id}>
                 <Title headingLevel="h4" className="pf-v5-u-mb-md">{`containers[${i}]`}</Title>
-                <ContainerConfigurationDescriptionList key={container.id} container={container} />
+                <ContainerConfigurationDescriptionList
+                    key={container.id}
+                    container={container}
+                    vulnMgmtBasePath={vulnMgmtBasePath}
```

Comment:
An observation, not a change request: seeing that component architecture separates base from complete path in contrast to `DeploymentOverview` below, what are pro and con as `platformComponent` becomes more widespread:
* property is base path of destination component, as here.
* property like `isPlatformComponent` itself, independent of cross-container destination:
    ```ts
    const isPlatformComponent =
        isFeatureFlagEnabled('ROX_PLATFORM_CVE_SPLIT') &&
        Boolean(deployment?.platformComponent);
    ```

Reply:
I went with the current approach because `ContainerImage` already had a conceptual dependency on the VM URL, due to it being hard coded internally. Instead of changing the conceptual dependency, the dependency was just moved up to be the responsibility of the parent. (Unlike in the case of `DecriptionListItem` below, which has all the information it needs to derive the URL internally.)

Using a general `isPlatformComponent` property could be beneficial in the future, if/when more aspects of the component's behavior become dependent on whether or not it is a platform component.